src/database/notifiers/slack_notifier.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
in the same sense, this would be something like
`web_team_slack_notifier.json`
```{type: 'SlackNotifier', settings: {}}``` 
then you can point `web_team_slack_notifier` this from component